docs/content/releases.md

```diff
@@ -6,6 +6,70 @@ doctypes: ["concept"]
+* HELM deployments can now set [custom environment variables with controller.env](https://github.com/nginxinc/kubernetes-ingress/pull/3326). Thanks to [Aaron Shiels](https://github.com/AaronShiels) for making this possible!
+* HELM deployments can now configure a [pod disruption budget](https://github.com/nginxinc/kubernetes-ingress/pull/3248) allowing deployments to configure either a minimum number or a maximum unavailable number of pods. Thanks to [Bryan Hendryx](https://github.com/coolbry95) for making this possible!
+* The NGINX Ingress Controller uses the latest OIDC reference implementation which now supports [access tokens for authorization](https://github.com/nginxinc/kubernetes-ingress/pull/3474) against protected resources. Thanks to [Shawn Kim](https://github.com/shawnhankim) for making this possible!
+* The default TLS secret is now optional. This ensures that TLS termination for not fall back to using the default TLS secret.
+
+FEATURES:
+* [3034](https://github.com/nginxinc/kubernetes-ingress/pull/3034) Allow extra args to be provided to the OIDC auth endpoint. Thanks to [Alan Wilkie](https://github.com/alanwilkie-finocomp).
+* [3474](https://github.com/nginxinc/kubernetes-ingress/pull/3474) Add access token support in the OIDC. Thanks to [Shawn Kim](https://github.com/shawnhankim).
+* [3326](https://github.com/nginxinc/kubernetes-ingress/pull/3326) Add support for custom environment variables on the Nginx Controller container. Thanks to [Aaron Shiels](https://github.com/AaronShiels).
+* [3527](https://github.com/nginxinc/kubernetes-ingress/pull/3527) Change controller.topologySpreadConstraints schema to array. Thanks to [Marco Londero](https://github.com/marcuz).
+* [3248](https://github.com/nginxinc/kubernetes-ingress/pull/3248) Add Pod disruption budget option to HELM based installations. Thanks to [Bryan Hendryx](https://github.com/coolbry95).
+* [3462](https://github.com/nginxinc/kubernetes-ingress/pull/3462) Add initial support for SSL termination for TransportServer.
+* [3451](https://github.com/nginxinc/kubernetes-ingress/pull/3451) Enable keepalive-time for healthchecks in VS and VSR.
+* [3560](https://github.com/nginxinc/kubernetes-ingress/pull/3560) Add support for load a pre-compiles AppProtect Policy Bundle.
+* [3632](https://github.com/nginxinc/kubernetes-ingress/pull/3632) Update nginx.org/ca secret type & crl field to IngressMTLS to support CRL.
+
+IMPROVEMENTS:
+* [3576](https://github.com/nginxinc/kubernetes-ingress/pull/3576) Replace deprecated github action command with environment variable. Thanks to [Jongwoo Han](https://github.com/jongwooo)
```

Comment:
```suggestion
```

I think we should remove this, it's not an improvement of the product

Reply:
Instead of removing it I'll move it to the HELM section. I think it's still worth calling out since this is an important feature for NJS applications.